meta-vehiclemodels/recipes-models/models/application_1.0.bb

```diff
@@ -1,31 +1,30 @@
-SUMMARY = "Ajout modeles"
+SUMMARY = "L'intégration de modéle dans l'image yocto"
 LICENSE = "CLOSED"
 
-S = "${WORKDIR}"
-
-SRC_URI = "https://github.com/yassinthabet/projet_PFE/raw/main/main.py;name=main \ 
-          https://github.com/yassinthabet/projet_PFE/raw/main/marque.json;name=marque \
-          https://github.com/yassinthabet/projet_PFE/raw/main/config.py;name=config \
-          https://github.com/yassinthabet/projet_PFE/raw/main/utils.py;name=utils \
-          https://github.com/yassinthabet/projet_PFE/raw/main/classes.names;name=classes \
-          https://github.com/yassinthabet/projet_PFE/raw/main/coco.names;name=coco \
-          file://darknet-yolov3.cfg \
-          file://demo.mp4 \
-          file://model.weights \
-          file://Nationality.pth \
-          file://yolov4.cfg \
-          file://yolov4.weights \
-          file://brand.pth \
+SRC_URI = "https://github.com/eyaderouich/pfe/raw/main/src/main.py;name=main \ 
+          https://github.com/eyaderouich/pfe/raw/main/src/marque.json;name=json \
+          https://github.com/eyaderouich/pfe/raw/main/src/config.py;name=config \
+          https://github.com/eyaderouich/pfe/raw/main/src/utils.py;name=utils \
+          https://github.com/eyaderouich/pfe/raw/main/src/classes.names;name=classes \
+          https://github.com/eyaderouich/pfe/raw/main/src/coco.names;name=coco \
+          https://github.com/eyaderouich/pfe/raw/main/src/darknet-yolov3.cfg;name=darknet \
+          https://github.com/eyaderouich/pfe/raw/main/Nationality.pth;name=nationality \
+          https://github.com/eyaderouich/pfe/raw/main/src/yolov4.cfg;name=yolov4 \
+          https://github.com/eyaderouich/pfe/raw/main/src/broker.cfg;name=configbroker \
           file://video1.mp4 \
```

Comment:
Create a new recipe for downloading the test videos, and test stuff and keep this recipe only for source code.
I prefer the videos aren't from local source, try compressing and loading them from drive or somewhere else ..